extensions/default/src/DicomWebProxyDataSource/index.js

```diff
@@ -0,0 +1,74 @@
+import { IWebApiDataSource } from '@ohif/core';
+import { createDicomWebApi } from '../DicomWebDataSource/index';
+
+function createDicomWebProxyApi(
+  dicomWebProxyConfig,
+  UserAuthenticationService
+) {
+  const { name } = dicomWebProxyConfig;
+  let dicomWebDelegate = undefined;
+
+  const implementation = {
+    initialize: async ({ params, query, url }) => {
+      let studyInstanceUIDs = [];
+
+      if (!url) url = query.get('url');
+
+      if (!url) {
+        console.error(`No url for '${name}'`);
+      } else {
+        const response = await fetch(url);
+        let data = await response.json();
+
+        // there seem to be a couple of variations of the case for this parameter
+        const queryStudyInstanceUIDs =
+          query.get('studyInstanceUIDs') || query.get('studyInstanceUids');
+        if (!queryStudyInstanceUIDs) {
+          console.error(`No studyInstanceUids in request for '${name}'`);
+        }
+        if (data.servers && queryStudyInstanceUIDs) {
```

Comment:
Shouldn't we throw here if `data.servers` don't exist? The rest of implementation relies on it